services/authenticator.js

```diff
@@ -12,9 +12,29 @@ const logger = require('./logger');
 const FORMAT_PASSWORD = /^(?=\S*?[A-Z])(?=\S*?[a-z])(?=\S*?[0-9])\S{8,}$/;
 
 function Authenticator() {
+  this.pathToLumberrc = `${os.homedir()}/.lumberrc`;
+
+  this.saveToken = token => fs.writeFileSync(this.pathToLumberrc, token);
+
+  this.isTokenCorrect = (email, token) => {
+    const sessionInfo = parseJwt(token);
+    if (sessionInfo) {
+      if ((sessionInfo.exp * 1000) <= Date.now()) {
+        logger.warn('You tried to use a token an expired token');
+        return false;
+      }
+
+      if (sessionInfo.data.data.attributes.email === email) {
+        return true;
+      }
+      logger.warn('You tried to use a token of another account');
```

Comment:
```suggestion
      logger.warn('Your token is invalid.');
```

I think we should stay vague in this catch-all case. BTW, I'm not sure this message is always true: a user that as a broken session data may be in that case too.

> _User type "p@ssw0rd"..._
> You tried to use a password of another account
> _Wohoo, I stole something and the system helps me understand how to do the next step!_

Reply:
Yeah but you need to have the token 🤔. You can't find it magically